src/porepy/geometry/half_space.py

```diff
@@ -9,35 +9,28 @@ def point_inside_half_space_intersection(
     n: np.ndarray, x0: np.ndarray, pts: np.ndarray
 ) -> np.ndarray:
     """
-    Find the points that lie in the intersection of half spaces (3D)
-
-    Parameters
-    ----------
-    n : np.ndarray, size 3 x num_planes
-        This is the normal vectors of the half planes. The normal
-        vectors is assumed to point out of the half spaces.
-    x0 : np.ndarray, size 3 x num_planes
-        Point on the boundary of the half-spaces. Half space i is given
-        by all points satisfying (x - x0[:,i])*n[:,i]<=0
-    pts : np.ndarray, size 3 x num_points
-        The points to be tested if they are in the intersection of all
-        half-spaces or not.
-
-    Returns
-    -------
-    out : np.ndarray, size num_points
+    Find the points that lie in the intersection of half spaces (3D).
+    
+    Examples:
+        >>> import numpy as np
+        >>> n = np.array([[0, 1], [1, 0], [0, 0]])
+        >>> x0 = np.array([[0, -1], [0, 0], [0, 0]])
+        >>> pts = np.array([[-1 ,-1 ,4], [2, -2, -2], [0, 0, 0]])
+        >>> half_space_int(n,x0,pts)
+        array([False,  True, False], dtype=bool)
+
+    Parameters:
+        n (shape = (3, num_planes)): This is the normal vectors of the half planes. The normal
+            vectors is assumed to point out of the half spaces. 
+        x0 (shape = (3, num_planes)): Point on the boundary of the half-spaces. Half space i is given
+            by all points satisfying (x - x0[:,i])*n[:,i]<=0
+        pts (shape = (3, num_points)): The points to be tested if they are in the intersection of all
+            half-spaces or not.
+
+    Returns:
         A logical array with length equal number of pts.
```

Comment:
```suggestion
        A logical array with ``shape=(np, )``.
```

Reply:
Here I was unsure if I should write ``shape=(np, )``, or ``len=np``.